examples/fscloud/main.tf

```diff
@@ -76,14 +52,14 @@ module "cbr_zone" {
 ##############################################################################
 
 module "postgresql_db" {
-  source              = "../../profiles/fscloud"
-  resource_group_id   = module.resource_group.resource_group_id
-  name                = "${var.prefix}-postgres"
-  region              = var.region
-  pg_version          = var.pg_version
-  key_protect_key_crn = module.key_protect_all_inclusive.keys["icd-pg.${var.prefix}-pg"].crn
-  resource_tags       = var.resource_tags
-  allowlist           = var.allowlist
+  source                     = "../../profiles/fscloud"
+  resource_group_id          = module.resource_group.resource_group_id
+  name                       = "${var.prefix}-postgres"
+  region                     = var.region
+  pg_version                 = var.pg_version
+  kms_key_crn                = var.kms_key_crn
+  existing_kms_instance_guid = var.existing_kms_instance_guid
+  resource_tags              = var.resource_tags
```

Comment:
This is missing `backup_encryption_key_crn` which has to be passed for FSCloud. It means the example needs to create a Key Protect instance and key as well